README.md

```diff
@@ -553,11 +553,27 @@ In version 1.23 the Kubernetes in-tree to container storage interface (CSI) volu
 
 An add-on is software that provides supporting operational capabilities to Kubernetes applications, but is not specific to the application. This includes software like observability agents or Kubernetes drivers that allow the cluster to interact with underlying AWS resources for networking, compute, and storage. [EKS Addons Guide](https://docs.aws.amazon.com/eks/latest/userguide/eks-add-ons.html)
 
-To enable the EBS CSI Driver (aws-ebs-csi-driver) set variables `enable_ebs_addon`and `create_addon_role` both  to true. The version of the driver addon is defined in the string variable `ebs_addon_version`
-To determine what versions of EBS CSI driver are supported use the command:
+The EBS CSI Driver (aws-ebs-csi-driver) by default is disabled. To enable set variables:
+```
+enable_ebs_addon = true
+create_addon_role = true
+ebs_addon_version = "v1.21.0-eksbuild.1"
+```
+To determine valid versions for variable `ebs_addon_version` use the command:
 ```
 aws eks describe-addon-versions --addon-name "aws-ebs-csi-driver" | jq -r '.addons[].addonVersions[].addonVersion'
 ```
+The EBS CSI Driver addon can be implemented successfully using **worker group launch templates**. The addon can be problematic when using the default **worker node groups** and is not recommended. For new installations set the following variables:
+```
+enable_worker_groups_launch_template = true
+enable_worker_group = false
+```
+For existing installations where `enable_worker_groups = true` only set the variable:
+```
+enable_worker_groups_launch_template = true
+```
+Once the addon is installed,  then perform [Transitioning from Worker Groups to Worker Groups Launch Templates](#transitioning-from-worker-groups-to-worker-groups-launch-templates). When completed the variable `enable_worker_group = false`
+
```

Comment:
Have you tried this? 

As in having a working cluster with nodes running with `enable_worker_groups = true` and then setting

```
enable_ebs_addon = true
create_addon_role = true
ebs_addon_version = "v1.21.0-eksbuild.1"
```

and that then failed.

Reply:
You are looking for a `enable_worker_groups` config and for now there's only a  `enable_worker_groups_launch_template` version.  I can build an `enable_worker_groups` env and test but will need to drop down to `1.22` to get it up and running.  I can let you know what I find.

Reply:
The reason I ask is that the documentation I commented on only would be relevant in the case I described, right? But if that case doesn't exist that documentation snippet would be  wrong.

Reply:
I thought you were identifying a scenario where you only wanted to use worker groups and not launch template.  The doc states that you have to set the launch template variable to true. I can include in the doc the current addon dependency is the required launch template if you think that would be beneficial. 
I can't get a new worker groups only environment built with the latest version (or previous version).  While debugging in the portal I found that the EC2 instances created alerts that an assigned security group can't be found. I suspect this might be influencing why you can't list the nodes with  the `kubectl get nodes` command.

Reply:
As far as I can see choosing worker groups or launch template has nothing to do with ebs csi. But in the current state of this module and how AWS is working worker groups doesn't work out of the box.

So if worker groups dosen't work I suppose it does make sense to change the default to use launch templates instead. But I don't see any reason to mention that in the documentation of ebs csi.

Reply:
Mårten, I removed the additional documentation as per your suggestion. I think we are in agreement but let me know if there's anything else outstanding.